client/my-sites/stats/feedback/style.scss

```diff
@@ -4,5 +4,45 @@
 	border-radius: 5px; // stylelint-disable-line scales/radii
 
 	padding: 20px;
-	margin: 32px;
+
+	@media (max-width: 782px) {
```

Comment:
We should probably use the predefined breakpoints. @dognose24 might have more information.